src/policy/policy.js

```diff
@@ -9,26 +9,12 @@ class Policy extends EventEmitter {
     super();
 
     this._isStrict = true;
-    if(isStrict === false){
+    if (isStrict === false){
```

Comment:
This is good to know:  
https://stackoverflow.com/a/359509/1551596  
(And even makes it more confusing :rofl:)